test/index.test.ts

```diff
@@ -219,6 +219,66 @@ describe('recipe parser', () => {
     });
   });
 
+  it('doesn\'t explode when 1 2 lb. chicken', () => {
+    expect(parse('1 2 lb. chicken')).to.deep.equal({
+      unit: '2 pound',
```

Comment:
Hmm, I'm a bit concerned about this because 2 isn't a unit that's convertible.

For example, what happens if someone uses this recipe parser to combine ingredients from multiple recipes to create a shopping list? If >2 recipes have chicken in it, how will it be able to combine: 
``` ts
{
      unit: '2 pound',
      ingredient: 'chicken',
      quantity: '1',
      minQty: '1',
      maxQty: '1'
}
```
with
```ts
{
      unit: 'pound',
      ingredient: 'chicken',
      quantity: '1',
      minQty: '1',
      maxQty: '1'
}
 ```
 
Another edge case this doesn't cover is for use-cases where the units need to be converted.
How do you convert `"2 pound" => kg`?

Reply:
I wonder if it be safer to omit a unit in this case? e.g.
```ts
{
      unit: null,
      ingredient: '2 lb. chicken',
      quantity: '1',
      minQty: '1',
      maxQty: '1'
}
```

Reply:
Ya, I understand the concern because a numeric value isn't really a unit of measure. However, I think the suggestion to not have a unit is actually worse than having a numeric part of the unit.

I haven't taken a look at the logic for combining ingredients, but I think it should be simple enough to do. First step would be assume that any unit without a numeric part has a 1 for the numeric part, the second would then be to convert units to the same base unit (adjusting any associated numeric portion correctly), finally sum up all numeric portions (including the assumed 1's) and then recombine. Which means that combining
```ts
{
      unit: '2 pound',
      ingredient: 'chicken',
      quantity: '1',
      minQty: '1',
      maxQty: '1'
}
```
with
```ts
{
      unit: 'pound',
      ingredient: 'chicken',
      quantity: '1',
      minQty: '1',
      maxQty: '1'
}
```
will result in
```ts
{
      unit: '3 pound',
      ingredient: 'chicken',
      quantity: '1',
      minQty: '1',
      maxQty: '1'
}
```
However, an issue comes into play when we have a quantity (and quantity ranges make it even worse). Should the number in the unit be multiplied by the quantity? Sometimes that is fine, but other times it doesn't make sense (in the case of cans or other uniform discrete items).

Maybe it would be better to take the number out of the unit and change the quantity accordingly. However, this still isn't perfect either. For example combining
```ts
{
      unit: '10.5 oz can',
      ingredient: 'cream of chicken soup',
      quantity: '1',
      minQty: '1',
      maxQty: '1'
}
```
with 
```ts
{
      unit: '10.5 oz can',
      ingredient: 'cream of chicken soup',
      quantity: '1',
      minQty: '1',
      maxQty: '1'
}
```
should not result in

Reply:
I think my exception for when there is a noun and a unit of measure in the unit field works out fine for combining logic, and conversion from pound -> kg or oz -> ml (or oz -> g) won't be too difficult as we use the number in the unit instead of the quantity.

Reply:
Ah, looking at it now my idea won't work, because there isn't ever multiple words in the unit, and it is incredibly difficult if not impossible to determine if a word after a unit of measure should be part of that "unit" or part of the ingredient.

Maybe everything I said above is beyond what is possible and not having any unit might actually be the better option now that I look at it again.